### ISSUE TITLE
@rokucommunity/bslint: not found in npm registry

### ISSUE THREAD

**Issue by heldrida** [rokucommunity/bslint#14]
Hi, first of all, thanks a lot for sharing the packages. Looks great!

the `@rokucommunity` organisation scope doesn't seem to exist. Either that or the organisation is set to `private`, see organisation tab [here](https://www.npmjs.com/~twitchbronbron?activeTab=orgs).

GET non-scoped package, is successful.
https://registry.npmjs.org/brighterscript

GET scoped package, fails.
https://registry.npmjs.org/@rokucommunity/bslint

Example of the current issue,

**Install (uses yarn, similar issue with npm although)**

```sh
yarn add @rokucommunity/bslint
```

**Error**

```sh
error An unexpected error occurred: "https://registry.npmjs.org/@rokucommunity%2fbslint: Not found".
```

Thank you!

**Comment by elsassph** [comment]
Hello, thanks for your interest, but the plugin hasn't been published yet :D

The brighterscript plugin API wasn't stable enough yet for that but it's getting there. Additionally it requires a bit of tuning to avoid false positives, and I thought it was light in linting options.

If you want to test it now:

- clone this git repository, then:
- `npm install && npm build && npm link`
- in your project: `npm link @rokucommunity/bslint`
- to unlink: `npm install`

**Comment by heldrida** [comment]
Ok, no problem! Sorry about that.
